site/docs/windows.md

```diff
@@ -119,6 +119,9 @@ C:\projects\bazel> bazel build //examples/cpp:hello-world
 C:\projects\bazel> bazel-bin\examples\cpp\hello-world.exe
 ```
 
+To use DLL files, see [this
```

Comment:
Nit: How about "To build and use dynamic library (DLL files)"

Reply:
SGTM, thanks!